resources/definitions/cartesio.def.json

```diff
@@ -0,0 +1,52 @@
+{
+    "id": "cartesio",
+    "name": "Cartesio",
+    "version": 2,
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "Scheepers",
+        "manufacturer": "Cartesio bv",
+        "category": "Other",
+        "file_formats": "text/x-gcode",
+        "has_materials": true,
+        "has_machine_materials": true,
```

Comment:
The material files are in https://github.com/ultimaker/fdm_materials . The original reason for that is to be able to share them between Cura and the UM3.